compute_tools/src/compute.rs

```diff
@@ -1937,19 +1937,46 @@ LIMIT 100",
         tokio::spawn(conn);
 
         // TODO: support other types of grants apart from schemas?
-        let query = format!(
-            "GRANT {} ON SCHEMA {} TO {}",
-            privileges
-                .iter()
-                // should not be quoted as it's part of the command.
-                // is already sanitized so it's ok
-                .map(|p| p.as_str())
-                .collect::<Vec<&'static str>>()
-                .join(", "),
-            // quote the schema and role name as identifiers to sanitize them.
-            schema_name.pg_quote(),
-            role_name.pg_quote(),
-        );
+
```

Comment:
This method is used via compute_ctl API, so should be fairly easy to test. Can we extend `test_role_grants` to validate this new change?

Reply:
Let me see if we are able to create read-replicas in this test framework.